cmd/server.go

```diff
@@ -109,7 +120,51 @@ func produceServerCmdRunFunc(o serverOperator, action int) cmdRunFunc {
 				log.Fatalf("Removing Server failed: %s", err)
 			}
 		}
+	case serverCreateAction:
+		return func(cmd *cobra.Command, args []string) {
+			ctx := context.Background()
+			cServer, err := client.CreateServer(ctx, gsclient.ServerCreateRequest{
+				Name:   serverName,
+				Cores:  cpu,
+				Memory: memory,
+			})
+			if err != nil {
+				log.Fatal("Create server has failed with error", err)
+			}
+			log.WithFields(log.Fields{
+				"server_uuid": cServer.ObjectUUID,
+			}).Info("Server successfully created")
 
+			if template != "" {
+				template, _ := client.GetTemplateByName(ctx, template)
+				cStorage, err := client.CreateStorage(ctx, gsclient.StorageCreateRequest{
+					Name:        string(serverName),
+					Capacity:    storage,
+					StorageType: gsclient.DefaultStorageType,
+					Template: &gsclient.StorageTemplate{
+						TemplateUUID: template.Properties.ObjectUUID,
+						Password:     plainPassword,
+						PasswordType: gsclient.PlainPasswordType,
+						Hostname:     hostName,
+					},
+				})
+				client.CreateServerStorage(
+					ctx,
+					cServer.ObjectUUID,
+					gsclient.ServerStorageRelationCreateRequest{
+						ObjectUUID: cStorage.ObjectUUID,
+						BootDevice: true,
+					})
+
+				if err != nil {
+					log.Error("Create storage has failed with error", err)
```

Comment:
This just logs the error (prints it to stderr) but continues normal execution. Is this intended?

Reply:
No it is not intended 😨 
It's noticed i will change it thanks!

Reply:
- Change log.Error ->  log.Fatalf
-  Exit status will be 1 from now on